yt/frontends/arepo/data_structures.py

```diff
@@ -0,0 +1,42 @@
+from yt.frontends.gadget.api import GadgetHDF5Dataset
+from yt.utilities.on_demand_imports import _h5py as h5py
+
+from .fields import \
+    ArepoFieldInfo
+
+
+class ArepoHDF5Dataset(GadgetHDF5Dataset):
+    _field_info_class = ArepoFieldInfo
+
+    def __init__(self, filename, dataset_type="arepo_hdf5",
+                 unit_base=None,
+                 smoothing_factor=2.0,
+                 index_order=None,
+                 index_filename=None,
+                 kernel_name=None,
+                 bounding_box=None,
+                 units_override=None,
+                 unit_system="cgs"):
+        super(ArepoHDF5Dataset, self).__init__(
+            filename, dataset_type=dataset_type, unit_base=unit_base,
+            index_order=index_order, index_filename=index_filename,
+            kernel_name=kernel_name, bounding_box=bounding_box,
+            units_override=units_override, unit_system=unit_system)
+        self.smoothing_factor = smoothing_factor
```

Comment:
Maybe add a comment explaining what this is, I realize it's documented in the narrative docs but it would also be nice to have an explanation in here as well